chart/dockerfiles/pgbouncer/Dockerfile

```diff
@@ -15,19 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 ARG ALPINE_VERSION="3.12"
-
 FROM alpine:${ALPINE_VERSION} AS builder
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
```

Comment:
```suggestion
SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
```

Reply:
Interesting mistake :)

Reply:
This hasn't been resolved, still says bash